### ISSUE TITLE
addRecipient incorrectly formats CONTACT_LISTS in XML request

### ISSUE THREAD

**Issue by creativeux** [Silverpop/node-engage#18]
Cannot add a recipient specifying the `contactLists` option because the child items of `CONTACT_LISTS` is malformed. Currently being added as an array index, not `CONTACT_LIST_ID`.

Can be fixed by adding the following to `add-recipient.js`

```js
if (typeof options.contactLists !== 'undefined') {
    subnode = [];
    for (key in options.contactLists) {
        subnode.push({
            CONTACT_LIST_ID: options.contactLists[key]
        });
    }
    params["CONTACT_LISTS"] = subnode;
}
```

**Comment by creativeux** [comment]
I have a branch I can push with a fix, but require permissions.

**Comment by creativeux** [comment]
[Pull request #20 open](https://github.com/Silverpop/node-engage/pull/20)

**Comment by ehutson** [comment]
The pull request has been merged and version 0.1.8 has been published.
Closing issue.
